go/vt/sqlparser/parse_test.go

```diff
@@ -2961,14 +2961,74 @@ func TestCreateTable(t *testing.T) {
 	},
 	}
 	for _, tcase := range testCases {
-		tree, err := ParseStrictDDL(tcase.input)
-		if err != nil {
-			t.Errorf("input: %s, err: %v", tcase.input, err)
+		t.Run(tcase.input, func(t *testing.T) {
+			tree, err := ParseStrictDDL(tcase.input)
+			if err != nil {
+				t.Errorf("input: %s, err: %v", tcase.input, err)
+				return
+			}
+			if got, want := String(tree.(*DDL)), tcase.output; got != want {
+				t.Errorf("Parse(%s):\n%s, want\n%s", tcase.input, got, want)
+			}
+		})
+	}
+
+	nonsupportedKeywords := []string{
```

Comment:
What does non supported mean? Do they work in MySQL?

The ones that work in MySQL should have a skipped test.

Reply:
i tried to add quick fixes for as many as i could, i haven't tested all of them with mysql

Reply:
i'll run them through mysql. what's the optimization trade-off for getting the long tail of mysql-supported statements?

Reply:
I wrote a script to check the remaining keywords -- mysql supports all of them. Added skipped tests to be more explicit.